tests/test_js_syntax.py

```diff
@@ -69,10 +69,10 @@ def test_blocks_evaluated():
     EVIL = 'eval(evilStuff)'
     BLOCKS = (
         'function foo() { %s; }',
-        'function foo() { %s; yield 1; }',
-        'function foo() { yield %s; }',
+        'function* foo() { %s; yield 1; }',
+        'function* foo() { yield %s; }',
         'var foo = function () { %s; }',
-        'var foo = function () { %s; yield 1; }',
+        'var foo = function* () { %s; yield 1; }',
```

Comment:
Those changes are needed for the tests to pass with this version of spidermonkey. Otherwise they raise a syntax error instead.

According to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/Legacy_generator_function, "The legacy generator function is a SpiderMonkey-specific feature, and will be removed at some point." (thanks @muffinresearch for digging that up).